config/policy.go

```diff
@@ -123,20 +130,20 @@ func (cfg *Policy) Validate() error {
 		cfg.FileSystems = defaultFilesystems
 	}
 
-	if cfg.Name == "" {
-		return errored.Errorf("Name is empty for policy")
+	size, err := cfg.CreateOptions.ActualSize()
+	if cfg.Backends.CRUD != "" && (size == 0 || err != nil) {
+		return errored.Errorf("Size set to zero for non-empty CRUD backend %v", cfg.Backends.CRUD).Combine(err)
 	}
 
-	if cfg.Backends.Mount == "" {
-		return errored.Errorf("Mount backend cannot be empty for policy %v", cfg)
+	if err := ValidatePolicy(cfg); err != nil {
+		return errors.ErrJSONValidation.Combine(err)
 	}
 
-	size, err := cfg.CreateOptions.ActualSize()
-	if cfg.Backends.CRUD != "" && (size == 0 || err != nil) {
-		return errored.Errorf("Size set to zero for non-empty CRUD backend %v", cfg.Backends.CRUD).Combine(err)
+	if cfg.Backends.Mount == "" { // backend should be defined and its validated
+		cfg.Backends = defaultDrivers[cfg.Backend]
```

Comment:
reading from maps has a secondary form:

```go
var, ok := myMap[myKey]
if ok {
  // key exists
}
```

This is really important here because we're not using pointers, and therefore this will always succeed.

If we don't have a default driver, perhaps we should throw an error? I'm not sure what the best option is here, ideas welcome.